core/src/templates/components/card/card.twig

```diff
@@ -33,26 +50,100 @@
   {%- set template_path_link = "@decanter/components/link/link.twig" -%}
 {%- endif -%}
 
+{%- if template_path_media is empty -%}
+  {%- set template_path_media = "@decanter/components/media/media.twig" -%}
+{%- endif -%}
+
 {# For variant where whole card is a link, suppress all other links within the component #}
 {%- if "su-card--link" in modifier_class -%}
   {%- set allow_links = false -%}
 {%- else -%}
   {%- set allow_links = true -%}
+  {% if card_allow_media_link == true %}
+    {%- set card_media_link = card_link -%}
+  {%- endif -%}
 {%- endif -%}
 
+<link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.8.2/css/all.css' type='text/css' media='' />
```

Comment:
Stylesheets are not allowed in twig templates.

Reply:
Just taking this out and all mention of the icon-font variant out till v6 :)